manila/share/drivers/netapp/dataontap/client/client_cmode.py

```diff
@@ -3273,8 +3273,19 @@ def unmount_volume(self, volume_name, force=False, wait_seconds=30):
         raise exception.NetAppException(msg % msg_args)
 
     @na_utils.trace
-    def delete_volume(self, volume_name):
+    def delete_volume(self, volume_name, force_delete):
         """Deletes a volume."""
+        if force_delete:
+            res = self.send_request(
+                'system-cli', [{'priv': 'advanced'}, {'args': [
+                    {'arg': 'volume'}, {'arg': 'destroy'},
+                    {'arg': '-vserver'}, {'arg': self.vserver},
+                    {'arg': '-volume'}, {'arg': volume_name},
+                    {'arg': '-force'}, {'arg': 'True'}
+                    ]}], enable_tunneling=False)
+            if res.get_child_content('cli-result-value') == '1':
```

Comment:
Can you please add a comment that explains that what exit code statuses mean here?
Would be also good to have an extra error log if something goes wrong.

Reply:
added comment as well as warning message. No error since we are anyway finishing delete operation